charts/frontend/templates/_helpers.tpl

```diff
@@ -145,6 +145,17 @@ rsync -az /values_mounts/ /backups/current/
     fieldRef:
       fieldPath: status.hostIP
 {{- end }}
+{{- if .Values.mailhog.enabled }}
```

Comment:
This outmost if-statement seems unnecessary.

Reply:
happened when i did a last minute change to remove a dedicated define in favor of just inline env variable. Had two redundant wrappers, thanks!